src/idiom308ex.fz

```diff
@@ -0,0 +1,29 @@
+ex308 is
```

Comment:
We have
```
  # convert this to a number using the given base.  If negative, add "-" as
  # the first character.
  #
  public as_string(base u32) String : character_encodings
```

in stdlib. Did you see this?

Reply:
no 🙈

It doesn't work for 0 in base 999999 but I guess this is rarely needed

Reply:
For base 999999 you would first need 999999 symbols. So no this is useless.